test/seeds/offchainSeeds/offchain.match2.seed.ts

```diff
@@ -298,6 +300,32 @@ export const match2Seed = async () => {
     },
   ])
 
+  await db.transaction().insert([
+    {
+      id: seededMatch2CancellationId,
+      api_type: 'match2',
+      transaction_type: 'cancellation',
+      local_id: seededMatch2Id,
+      state: 'submitted',
+      created_at: exampleDate,
+      updated_at: exampleDate,
+      hash: transactionHash,
+    },
+  ])
+
+  await db.transaction().insert([
+    {
+      id: seededMatch2CancellationId2,
+      api_type: 'match2',
+      transaction_type: 'cancellation',
+      local_id: seededMatch2Id,
+      state: 'submitted',
+      created_at: exampleDate,
+      updated_at: exampleDate,
+      hash: transactionHash,
+    },
+  ])
+
```

Comment:
```ts
(async (n) => {
  return new Array(n).fill({}).map((el) => db.transaction().insert([
    {
      id: generateRandomUUID(),
      api_type: 'match2',
      transaction_type: 'cancellation',
      local_id: generateLocalId(),
      state: 'submitted',
      hash: transactionHash,
    },
})()
```

mm... since we are starting to repeat

Reply:
Provided I add this, How will I use it later on in the tests when I am trying to retrieve specific records by referring to them by their specific UUIDS -> should I put the newly generated UUIDs in an array - import this array to my test file and just pull uuids from there? -> But I presume that I cannot assert for the what the body contains then